bibliopixel/project/types/defaults.py

```diff
@@ -0,0 +1,12 @@
+from . import channel_order, color, duration, gamma, ledtype
+
+FIELD_TYPES = {
+    'channel_order': channel_order,
```

Comment:
Costs nothing to have this.  And as you know, I prefer more explicit names that don't need documentation...